app/src/androidTest/java/ch/epfl/sdp/peakar/points/ComputePOIPointsTest.java

```diff
@@ -149,4 +168,71 @@ public void getPOIPointsTest() {
         assertFalse(ComputePOIPoints.getFilteredPOIsOutOfSight().containsKey(filteredLabeledInSight));
         assertFalse(ComputePOIPoints.getFilteredPOIsOutOfSight().containsKey(LabeledInLineOfSight));
     }
+
+    /* Test if the offline content is loaded correctly */
+    @Test
+    public void loadPOIsFromFileTest() {
+
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(mContext);
+
+        // Activate offline mode
+        // Offline mode not activated, reset shared preference
+        prefs.edit()
+                .putBoolean(mContext.getResources().getString(R.string.offline_mode_key), true)
+                .apply();
+
+
+
+        BoundingBox mBoundingBox = userPoint.computeBoundingBox(GeonamesHandler.DEFAULT_RANGE_IN_KM);
+        System.out.println(mBoundingBox.getCenterWithDateLine());
+
+        List<POIPoint> mPoiPoints = new ArrayList<>();
+        POIPoint pointOne = new POIPoint("pointOne", 45.802540, 6.850330, 3000);
+        POIPoint pointTwo = new POIPoint("pointTwo", 45.802536, 6.850326, 5000);
```

Comment:
Use constants here instead of meaningless numbers

Reply:
This locations are used just one time in the code, I don't think it is needed